assets/sass/06-components/widgets.scss

```diff
@@ -11,6 +11,10 @@
 	ul {
 		list-style-type: none;
 		padding: 0;
+
+		&.sub-menu {
+			margin-left: var(--primary-nav--padding);
```

Comment:
@carolinan or @melchoyce Do you think it's worth to introduce a new variable in this case, e.g. `--global--spacing-vertical` or using `margin-left: calc(0.66 * var(--global--spacing-unit));` directly? To me, using a variable for the margins of list elements which is called `--primary-nav--padding` is rather confusing. We're neither adjusting the primary navigation here nor do we adjust the padding. 🤔

Reply:
I agree it might be confusing to use that variable in a whole different context, as it's clearly referring to the navigation, not the widgets.

Reply:
I will update this with new variable

Reply:
@nielslange @luminuu  should we use `--global--spacing-submenu` instead of `--global--spacing-vertical` because margin-left is related to submenu class. Or we just have to go with `margin-left`?

Reply:
I wonder if `--global--spacing-submenu` isn't too narrow scoped. I'd probably search for all `margin-left:` appearances in the project and look up what definitions they're using. 

My intention in https://github.com/WordPress/twentytwentyone/pull/235#discussion_r499130530 was less about creating an additional variable but more to use a speaking variable. And using `...--padding` to adjust a `margin` doesn't sound right to me.

Reply:
Yup @nielslange Got your point. Let's wait for @carolinan What's her point of view.

Reply:
Using --padding to adjust a margin doesn't sound right to me either. 
This is also used for margin in the primary navigation that are not inside a widget area.

Reply:
Can we create `--global--spacing-widget-menu` variable? So, It will be consistent across all the widget navigation menu.

Reply:
I'm not sure how often one would need to change the spacing for the menu?
This has been delayed because I got stuck on details. It could be merged without the menu styles, because it fixes the other widgets.

Reply:
Ok Sure I will remove style of menu. And will create a separate issue for nav menu widget style.